src/component/base/DraftEditorProps.js

```diff
@@ -106,6 +106,7 @@ export type DraftEditorProps = {
   ariaMultiline?: boolean,
   ariaOwneeID?: string,
   webDriverTestID?: string,
+  id?: string;
```

Comment:
Populate this new property in `examples/draft-0-10-0/customStyles/customStyles.html` so we can test it is working before creating a new version of the package

Reply:
Id gets indeed sent through to the content editable, I have increased the version to 0.11.7-internal.8